scripts/webpack/frontend.js

```diff
@@ -43,7 +43,7 @@ const common = ({ platform, page = '' }) => ({
                                 '@babel/preset-typescript',
                                 '@babel/preset-react',
                                 [
-                                    '@babel/preset-env',
```

Comment:
Talked in person, but just updating for prosperity:

I think we still need to compile with both but serve preset-env to those without module support and preset-modules to those with:

From the docs:
```
<!-- transpiled with preset-modules: -->
<script type="module" src="modern.js"></script>
<!-- transpiled with preset-env: -->
<script nomodule src="legacy.js"></script>
```

We likely need to loop over each `preset-module` and `preset-env` transpiled script here: https://github.com/guardian/dotcom-rendering/blob/8ad14b0cd290175a625e82c1b1d392508a40c7ee/src/web/server/htmlTemplate.ts#L40 and add the relevant `type="module"` or `nomodule`